common/go/xpacket/packet.go

```diff
@@ -0,0 +1,52 @@
+package xpacket
```

Comment:
Why not just move this file from tests/go/common/helpers.go here and update the references in other places? It seems like the right time to do this rather than duplicating it.